thirdparty/scripts/setup.sh

```diff
@@ -67,3 +67,8 @@ bash "$TP_SCRIPT_DIR/build_ui.sh"
 # rDSN (optional)
 ##############################################
 # bash "$TP_SCRIPT_DIR/build_rdsn.sh"
+
+##############################################
+# modin
+##############################################
+bash "$TP_SCRIPT_DIR/build_modin.sh"
```

Comment:
I'm not sure if this is the right way to do this, since it seems like we're deprecating this file, though I'm not sure. @chuxi can you comment?

Reply:
I am certainly willing to move this, but I am not sure where it will go.